tendenci/apps/events/views.py

```diff
@@ -3871,6 +3872,55 @@ def registrant_roster(request, event_id=0, roster_view='', template_name='events
         'total_checked_in': total_checked_in})
 
 
+@is_enabled('events')
+@login_required
+def digital_check_in(request, registrant_id, template_name='events/reg8n/checkin.html'):
```

Comment:
Can we add some permission check for digital_check_in? Right now, looks like all logged-in users can check in and check out anybody.

    if not has_perm(request.user, 'events.view_registrant'):
        raise Http403

Reply:
yes, looks like I missed that... will add it & push the code back up

Reply:
updated